gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/zosmf/ZosmfService.java

```diff
@@ -221,15 +220,23 @@ public String getZosmfRealm(String infoURIEndpoint) {
     /**
      * Verify whether the service is actually accessible.
      *
+     * Note: This method uses getURI, it's also verifying eureka registration
+     *
      * @return true when it's possible to access the Info endpoint via GET.
      */
     public boolean isAccessible() {
         final HttpHeaders headers = new HttpHeaders();
         headers.add(ZOSMF_CSRF_HEADER, "");
 
-        String infoURIEndpoint = getURI(getZosmfServiceId()) + ZOSMF_INFO_END_POINT;
-        log.debug("Verifying z/OSMF accessibility on info endpoint: {}", infoURIEndpoint);
+        String infoURIEndpoint = "";
+        try {
+            infoURIEndpoint = getURI(getZosmfServiceId()) + ZOSMF_INFO_END_POINT;
+        } catch (ServiceNotAccessibleException e) {
+            log.debug("URI not available because z/OSMF is not registered or wrong URL in Discovery Service");
```

Comment:
Make it sense to mentioned z/OSMF service ID?

Reply:
I can, but given the scenarios where this is printed, it will be covered by more detailed messages

Reply:
done